RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -312,7 +312,14 @@ void finalizeMaximize()
 	}
 	if(!in_zelda() && get_property(getMaximizeSlotPref($slot[weapon])) == "" && !maximizeContains("-weapon") && my_primestat() != $stat[Mysticality])
 	{
-		addToMaximize("effective");
+		if (my_class() == $class[Seal Clubber] && auto_my_path() == "G-Lover")
+		{
+			addToMaximize("club");
+		}
+		else
+		{
+			addToMaximize("effective");
+		}
```

Comment:
nice about the club. although there are occasions where the lightsaber is better than the club for a SC for the most part they really do prefer clubs

And it even checks to make sure primestat is not mysticality, nice!